### DISCUSSION TITLE
关闭单独网站的 WAF，但是还是会拦截

### DISCUSSION THREAD

**Discussion by Alexzzz2008** [1Panel-dev/1Panel#6065] [Q&A]
### 联系方式

1

### 1Panel 版本

1.10.13

### 问题描述

在waf的网站设置里 关闭了 当前网站的waf

但是实际操作中 还触发了频率限制并短暂屏蔽了Ip

然后在全局了 关闭了waf 才正常。

![PixPin_2024-08-02_23-43-43](https://github.com/user-attachments/assets/bce3651e-249f-42de-b737-bd41b1061597)

不知道是不是 bug 

### 重现步骤

1

**Comment by zhengkunwang223** [comment]
感谢反馈，我们先验证一下

**Comment by zhengkunwang223** [comment]
没有复现 能否提供更多信息？或者提供环境我们测试一下
